src/biogeophys/SurfaceAlbedoMod.F90

```diff
@@ -1051,7 +1034,24 @@ subroutine SurfaceAlbedo(bounds,nc,  &
             fcansno(bounds%begp:bounds%endp), surfalb_inst)
 
     else
-
+       
```

Comment:
Ahh, so this is another place where this caught this pattern that was being done for FATES, and now you have it only done outside of FATES.

Reply:
yes